.github/workflows/LT_CI.yml

```diff
@@ -1,11 +1,11 @@
-name: Build OpenEXR iOS static lib
+name: Build OpenEXR iOS static framework
 on: [push, pull_request]
 env:
-  DEVELOPER_DIR: /Applications/Xcode_11.5.app/Contents/Developer
+  DEVELOPER_DIR: /Applications/Xcode_13.3.1.app/Contents/Developer
```

Comment:
I think we can just delete this line

Reply:
Not really, in a few weeks Xcode 13.4 will be the default

Reply:
Ok, then I'll keep this line, thanks for confirming